### ISSUE TITLE
BUG in frontend (bottom bar) "colab-status-bar" (notebook included)

### ISSUE THREAD

**Issue by leinaxd** [googlecolab/colabtools#3109]
TL:DR Notebook at the end
-----------
**This is a Frontend bug report.**
While running a cell, the bottom status bar is bug opened.
- The items in this bar may fill the entire screen.
- You cannot close this bar.

**The expected behaviour is:**
- Getting just ONE line of status bar.
- Be abble to close this bar

**Reproducible Steps**
1. Run something.
2. Read Wikipedia for 10 min. (change chrome tab focus)
3. After 10mins, come back and surprise: the status bar turned gray and its painly bugged!

**Additional context**
- i'm using Chrome (up to date) with Kubuntu also i use zoom of 150%
- i've made a Notebook 
https://colab.research.google.com/drive/1kRe6Kk_VeliQbsD3gLJ9iIfb0fRdKOD0?usp=sharing

**Comment by cperry-goog** [comment]
This is a great bug report: thank you!

We've heard about this here and there but have never been able to reproduce - I can't get this to repro on my chromebox + chrome. I'm wondering if there's a conflicting browser extension that might be causing this for you.

**Comment by cperry-goog** [comment]
This is indeed elusive, I'm hoping one day we can catch it...

Thanks for helping, let me know if you have success.

**Comment by leinaxd** [comment]
In the end, you were right. 
The bug came from the "Super Dark Mode" extension which is not turned off correctly.

thanks for the support
